### ISSUE TITLE
example

### ISSUE THREAD

**Issue by johnnyagerard** [dotnet/docs#6417]
When compiling the bus code sample, I have the following values:
-31.14 minutes
-65.53 minutes

Why do you get whole numbers values 6.00 and 31.00 ?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: aa3ed1c5-1f79-9234-beec-b5771c3fb3c0
* Version Independent ID: ac4f6361-70a7-fb43-101e-c12044f60896
* Content: [Static Constructors (C# Programming Guide)](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/static-constructors)
* Content Source: [docs/csharp/programming-guide/classes-and-structs/static-constructors.md](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/classes-and-structs/static-constructors.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by johnnyagerard** [comment]
So I guess you used the Milliseconds method instead of TotalMilliseconds.

**Comment by BillWagner** [comment]
Looking at the sample and the output, I believe your previous comment is correct: the output uses the `Milliseconds` property, while the code uses the `TotalMilliseconds` property. 

I think the best fix is to update the sample to use the `Milliseconds` property.

I added this to our backlog, and added the "up for grabs" label.

**Comment by johnnyagerard** [comment]
@svick No I did not get any negative value. I just wrote a list of values which is indeed ambiguous.

@BillWagner I was wrong. I think the example is correct. It just randomly produced whole values in your example and I could not believe it was random. The chances of this happening are increased by the fact that the value is rounded off at 2 decimals.
And if you get a first whole value, the program will "sleep" exactly 25 ms, and you will have almost certainly a second whole value.
This is pretty funny no?
